syncd/syncd.cpp

```diff
@@ -40,6 +40,7 @@ void exit_and_notify(int status)
     exit(status);
 }
 
+#ifdef BRCMSAI
```

Comment:
I didn't realize that SAI_SWITCH_ATTR_CUSTOM_RANGE_BASE is not supported. Will merge it.